lease/lessor.go

```diff
@@ -161,13 +168,24 @@ func (le *lessor) Attach(id LeaseID, items []leaseItem) error {
 	return nil
 }
 
+func (le *Lessor) Recover(b backend.Backend, dr DeleteableRange) {
```

Comment:
`func Recover(lessorID uint8, b backend.Backend, dr DeleteableRange) Lessor` might be cleaner since the object is getting clobbered into essentially a fresh object